common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/WebSocketCoreImpl.kt

```diff
@@ -220,19 +226,9 @@ internal class WebSocketCoreImpl(
 
     override suspend fun sendBytes(data: ByteArray): Boolean? {
         return if (connect()) {
-            // TODO Timeout can never happen here and it should be removed.
-            // since send returns immediately and we can't timeout a synchronized call
-            // but we should check the return of send
```

Comment:
Why did you decide not to check the return of send here?

Reply:
It is sent to the caller I didn't want to change that. I should probably still log it.

Reply:
While looking at the usage of this function nothing checks that it went fine. What is your take here? I think it would make sense to create an issue.

> This method returns true if the message was enqueued. Messages that would overflow the outgoing message buffer (16 MiB) will be rejected and trigger a graceful shutdown of this web socket. This method returns false in that case, and in any other case where this web socket is closing, closed, or canceled.

Today I don't think we will ever reach 16MiB since we only use it for voice. But for future usage it might be important to check the size too.

Reply:
An issue to track the possible improvement would be fine, but I don't see this becoming an issue anytime soon.